common/src/Utilities/Linux/LnxThreads.cpp

```diff
@@ -113,9 +117,13 @@ void Threading::pxThread::_platform_specific_OnCleanupInThread()
 
 void Threading::pxThread::_DoSetThreadName( const char* name )
 {
+#if defined(__linux__)
 	// Extract of manpage: "The name can be up to 16 bytes long, and should be
 	//						null-terminated if it contains fewer bytes."
 	prctl(PR_SET_NAME, name, 0, 0, 0);
+#elif defined(__unix__)
+	pthread_set_name_np(pthread_self(), name);
```

Comment:
`pthread_set_name_np` - FreeBSD/OpenBSD
`pthread_setname_np` - Linux/NetBSD

Great fun ;) But yeah could be worth changing later.